### PR TITLE
valetudo-mapper addon

### PR DESCRIPTION
**Pull request by apoc4lyps**
add the valetudo-mapper addon for Valetudo RE (https://github.com/rand256/valetudo)

### CODE REVIEW

**Comment by apoc4lyps** [comment]
cant get the armhf, armv7 and aarch64 build running. Any ideas, otherwise I would remove the PR and host it myself.

**Comment by Poeschl** [comment]
I will try to put some output during the install to get the build time a little extended. How long it takes to get build on your machine?

In the worst case the image key in the config can be removed, which causes Hassio to build it itself on the raspberry.

File: valetudo-mapper/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ENV VALETUDOMAPPER_VERSION 0.2.0
+ENV COMMIT_SHA '02655ed8f2e1c81d6b1798fd63c9b7c0ef70e382'
+
+RUN apk add --no-cache git=2.20.1-r0 npm=10.14.2-r0
+# hadolint ignore=DL3003
+RUN git clone https://github.com/rand256/valetudo-mapper -b master /app && \
+    cd app && git checkout $COMMIT_SHA
+
+WORKDIR /app
+RUN npm install
```
[the remark this answers is not in the document]
Comment (Poeschl) [reply valetudo-mapper/Dockerfile]: I also tested the build on the failing plattforms. Even after an hour no progress was made behind the post install point.

Are you sure the valetudo-mapper is supported on `armhf`, `armv7` and `aarch64` ?

[the remark this answers is not in the document]
Comment (apoc4lyps) [reply valetudo-mapper/Dockerfile]: there were some issues with the alpine image and the nodejs version on arm. With the last commit it now works.

**Comment by Poeschl** [comment]
Thanks a lot for your contribution 👍

**Pull request merged**